proposals/emptyset/es003.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers } from 'ethers'
+import { EMPTYSET_CONTRACTS } from './contracts'
+import { Proposal } from '../../test/testutil/govern'
+
+const PROPOSAL_TEXT = `# Enabling Cheaper L1 / L2 Wrapping
```

Comment:
We should update this text 🙏

Reply:
updated 🔥, we might need to update this again per forum feedback as well.